airflow/sensors/filesystem.py

```diff
@@ -57,6 +58,9 @@ def poke(self, context):
 
         for path in glob(full_path):
             if os.path.isfile(path):
+                mod_time = os.path.getmtime(path)
+                mod_time = datetime.fromtimestamp(mod_time).strftime('%Y%m%d%H%M%S')
+                self.log.info(f'Found File {full_path} last modified: {mod_time}')
```

Comment:
Since the logging takes place inside the loop, this should probably log `path`
```suggestion
                self.log.info(f'Found File {path} last modified: {mod_time}')
```